cudamapper/tests/Test_CudamapperOverlapper.cpp

```diff
@@ -11,7 +11,7 @@
 #include "gtest/gtest.h"
 #include <string>
 #include <vector>
-#include "../include/claragenomics/cudamapper/overlapper.hpp"
```

Comment:
Is our convention that we take global includes over local ones? These two includes have distinctly different meanings if you've got both a global install (e.g. in /usr/local) and a local copy of the repo.

Reply:
the cmake setup adds the public include paths to the compiler, so we use the headers using this convention as they get picked up from the include paths.

for global vs local copy, I'd say in those situations the include lists should be setup to have only one of them, and not both. otherwise it gets pretty confusing which one is to use where and a potential recipe for nasty bugs...